news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -18,106 +18,13 @@ tags:
 In March 2020, the product team at PrestaShop, in charge of building and prioritizing new features, decided to work with a new tool: [Productboard](https://www.productboard.com). It helps product managers to create relevant roadmaps by evaluating features and tracking their development.
 
 Productboard made two things possible for the team:
-- Having a team interface to centralize all feedback, organize them, define features, prioritize them, and follow their development.
+- Having a team interface to centralize all feedback, organize them, define features, and prioritize them.
 - Having a public space, where the team is able to publicly present these features and to collect feedback about them. **And that is what this article is about.**
 
-So the team has created this space called [**PrestaShop Feedback**](https://feedback.prestashop.com) where anyone can comment, and vote for the next features in the roadmap, whether you are a merchant, a developer, an agency, a seller on the [Addons Marketplace](http://addons.prestashop.com/),...
+So the team has created this space called [**PrestaShop Feedback**](https://feedback.prestashop.com) where anyone can comment, and vote for the next features the team is working on, whether you are a merchant, a developer, an agency, a seller on the [Addons Marketplace](http://addons.prestashop.com/),...
 This space is accessible at [feedback.prestashop.com](https://feedback.prestashop.com).
 
-Before explaining how to use the **PrestaShop Feedback** website in the [second part](#how-to-use-prestashop-feedback), here is some context and information about how product development is done at PrestaShop. We’ll try to keep it short here, feel free to ask the team if you want to learn more about it.
-
-If you just want to learn how to use PrestaShop Feedback, [click here](#how-to-use-prestashop-feedback)
```

Comment:
There was in the past a feedback.prestashop.com based on Uservoice. Why not just tell in the introduction that we are moving from Uservoice to ProductBoard to collect user feedback?

This article is about collecting feedback, but not about how the team actually handle them, nor about the roadmap.